src/Shared/runtime/ReadMe.SharedCode.md

```diff
@@ -24,7 +24,7 @@ dotnet/AspNetCore code paths:
 - `PS D:\github\runtime\src\libraries\System.Net.Http\tests\UnitTests> dotnet build /t:test`
 
 ## Building dotnet/AspNetCore code:
-- https://github.com/dotnet/AspNetCore/blob/master/docs/BuildFromSource.md
+- https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md
```

Comment:
This file must be updated in runtime as well or the bot will overwrite this change.

Reply:
Thanks @Tratcher❕ See dotnet/runtime#47306